packages/helm-charts/celo-fullnode/values.yaml

```diff
@@ -0,0 +1,85 @@
+# Default values for celo-fullnode.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+fullnameOverride: false
+
+service:
+  type: LoadBalancer
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # requests:
+  #  cpu: 100m
+  #  memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+storage:
+  enable: true
+  # AKS default storageclasses:
+  #   - default for Azure Standard storage (HDD)
+  #   - managed-premium for Azure Premium Disks (SSD)
+  storageClass: default
+  accessModes: ReadWriteOnce
+  size: 20Gi
+
+geth:
+  image:
+    repository: us.gcr.io/celo-testnet/geth
+    tag: 1b40b25d315bfcd792138e288ea61351d6c44d09
+    imagePullPolicy: IfNotPresent
+  resources:
+    requests:
+      memory: "256Mi"
+      cpu: "500m"
+    limits:
+      memory: "512Mi"
+      cpu: "1000m"
+  debug: false
+  syncmode: full
+  gcmode: full
+  ping_ip_from_packet: false
+  in_memory_discovery_table: false
+  verbosity: 2
+  # Azure static ips
+  public_ips:
+  - 1.2.3.4
```

Comment:
Should these be internal IP addresses? How does the Oracle service reach the load balancer?

Reply:
These are static public ips in the same resource group as the aks nodes (also can be used same approach in other cloud providers like gce or aws), and after creating them, they are assigned to a public Load Balancer using the property `loadBalancerIP`. We use this approach to expose the celo-blockchain service using the load balancer ip, and we need to create the ips in advance because the celo-blockchain pod/process needs to know its assigned ip in advance to use in the `--nat` option.